client/app/helpers/format-date.coffee

```diff
@@ -1,10 +1,11 @@
 `import Ember from 'ember'`
 
-formatDate = (date, options) ->
+formatDate = (date, format) ->
```

Comment:
yup, right now ember helpers expect the optional argument to be sent explicitly like `{{format-date blahDate format="L"}}`

thanks for checking  :+1: